purchases/src/main/kotlin/com/revenuecat/purchases/utils/RateLimiter.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases.utils
+
+internal class RateLimiter(val maxCalls: Int, val periodSeconds: Int) {
+    companion object {
+        const val MILLISECONDS_PER_SECOND = 1000
+    }
+
+    private val maxCallInclusive: Int
+        get() = maxCalls + 1
```

Comment:
Hmm I think this can be
```suggestion
    private val maxCallInclusive = maxCalls + 1
```